add a TLS server (#13)
Kong/go-echo: README.md

This repo is a fork of https://github.com/cjimti/go-echo maintained by Kong. The main purpose of this fork is to have multi-arch images with both amd64 and arm64 support.

## Quick Start Guide

Run `go-echo` to start a TCP server listening port on 1025. The server will return a welcomet message to the client and echo what it received from client. Set environment variable `TCP_PORT` to configure another port to listen on.

> Set `TLS_PORT` to the port to listen on, `TLS_CA_CERT_FILE` to path of CA certificat file, `TLS_CERT_FILE` and `TLS_KEY_FILE` to paths of certificate-key pair to start a TLS server doing the same thing.

Reviewer: ```suggestion
In order to run the server as a TLS server, set `TLS_PORT` to the port to listen on, `TLS_CA_CERT_FILE` to path of CA certificate file, `TLS_CERT_FILE` and `TLS_KEY_FILE` to paths of certificate-key pair.
```
